timely_beliefs/beliefs/classes.py

```diff
@@ -239,6 +289,8 @@ def query(
         :param belief_before: only return beliefs formed before this datetime (inclusive)
         :param belief_not_before: only return beliefs formed after this datetime (inclusive)
         :param source: only return beliefs formed by the given source or list of sources (pass their id or name)
+        :param sensor_cls: set to your own DBSensor class implementation if it uses a different table name
```

Comment:
Can't we get the sensor class from `sensor.__class__`? (and the same for `source.__class__`)

Reply:
Just saw that you don't expect the sources as an object (why not?), so yes then you need to ask which type the objects really should have.

Also, that you're typing the `sensor` parameter as `DBSensor`, which is typically not used by libraries implementing their sensor using the `SensorDBMixin`. Maybe that is a better expected type here...

Reply:
That's a very neat solution, thanks.